cis/schema.json

```diff
@@ -29,54 +29,54 @@
+            "description": "If false, all access for this user should be denied. Profile consumers MUST check this value and disregard the remainder of the profile if false."
         },
         "lastModified": {
             "type": "string",
             "format": "date-time",
-            "description": "(Authoritative) Date of last profile modification"
+            "description": "Date of last profile modification"
         },
         "created": {
             "type": "string",
             "format": "date-time",
-            "description": "(Authoritative) Date of profile creation"
+            "description": "Date of profile creation"
         },
         "userName": {
             "type": "string",
-            "description": "UNKNOWN"
+            "description": "Human-readable login name or identifier for this user. For a unique identifier, use `user_id` instead."
         },
         "displayName": {
             "type": "string",
-            "description": "(Informational) Human-readable full name for this user"
+            "description": "Human-readable full name for this user"
         },
         "firstName": {
             "type": "string",
-            "description": "(Informational) Human-readable first name (surname) for this user"
+            "description": "Human-readable first name (surname) for this user"
         },
         "lastName": {
             "type": "string",
-            "description": "(Informational) Human-readable last (family) name for this user"
+            "description": "Human-readable last (family) name for this user"
         },
         "preferredLanguage": {
             "type": "string",
-            "description": "(Informational) Preferred language for this user, as a locale code such as `en_US`"
+            "description": "Preferred language for this user, as a locale code such as `en_US`"
         },
         "primaryEmail": {
             "type": "string",
             "format": "email",
-            "description": "(Informational) The primary email address at which the user may be reached.  Profile consumers may use this email to communicate with the user, but should not use it as a stable identifier for the user."
+            "description": "The primary email address at which the user may be reached.  Profile consumers may use this email to communicate with the user, but should not use it as a stable identifier for the user."
         },
         "emails": {
             "type": "array",
-            "description": "(Informational) All email addresses associated with this user's profile. Order is undefined.",
+            "description": "All email addresses associated with this user's profile. Order is undefined.",
             "items": {
                 "type": "object",
                 "properties": {
```

Comment:
It might be worth adding `required` and `additionalProperties` here (and for the other arrays of sub-objects), too.